### PR TITLE
SillyKoalaHugs.BabySmash version 1.0

### PR DESCRIPTION
**Pull request by AwesomeKalin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97191)

### CODE REVIEW

**Comment by AwesomeKalin** [comment]
The installer does work for me. I don't know why it's failing

**Comment by AwesomeKalin** [comment]
Why isn't it starting?

**Comment by AwesomeKalin** [comment]
I've started two pipelines and this still isn't running! I think this may need to be manually stopped

**Comment by AwesomeKalin** [comment]
This is going to fail. I can tell from the manifest. I'm going to fix it

File: manifests/s/SillyKoalaHugs/BabySmash/1.0/SillyKoalaHugs.BabySmash.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SillyKoalaHugs.BabySmash
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: SillyKoalaHugs
+PackageName: Baby Smash
```
Comment (stephengillie) [inline manifests/s/SillyKoalaHugs/BabySmash/1.0/SillyKoalaHugs.BabySmash.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://sillykoala.eu.org/
PackageName: Baby Smash
PackageUrl: https://sillykoala.eu.org/baby-smash/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**